tests/test_simple_put.cpp

```diff
@@ -495,7 +505,17 @@ void test_simple_put_skip(void **unused)
 
   // load first time should return uploaded
   std::string put_status;
-  ITransferResult * results = agent.execute(&putCommand);
+
+  ITransferResult * results = NULL;
+  try
+  {
+    results = agent.execute(&putCommand);
+  }
+  catch (...)
+  {
+    assert_string_equal("", sfstmt->error.msg);
```

Comment:
so why no error message even if an exception is raised here?

Reply:
Without this change, if the query execution failed, exception is raised and terminate the process so following test cases won't be run.
This change is to catch the exception, since the error message is not empty when query fails, the assertion will fail this test case with error message out in the test output, and continue with following test cases.

Reply:
Sorry, I'm confused. I understand the statement, "Without this change, if the query execution failed, exception is raised and terminate the process so following test cases won't be run."

But why this assert ensures the error message is an empty string? Should not the error context have some message instead of stdout?

https://api.cmocka.org/group__cmocka__asserts.html#ga3db7885581668648088fc4eb4cee6d4f